drivers/unix/ip_unix.cpp

```diff
@@ -48,6 +48,9 @@
       #include <iphlpapi.h>
       #undef _WIN32_WINNT
       #define _WIN32_WINNT 0x0600 // Reenable Vista API
+      #ifndef IPV6_V6ONLY
+        #define IPV6_V6ONLY 27 // Apparently, IPV6_V6ONLY flag is missing too in this version...
```

Comment:
Didn't work, it's not the right place for it :)

Just keep it in `socket_helpers.h` with a comment that it works around a mingw-w64 < 3.0 missing definition, and that the value comes from `<ws2ipdef.h>` of the Windows SDK.